armi/physics/neutronics/parameters.py

```diff
@@ -655,8 +655,6 @@ def _getNeutronicsBlockParams():
             categories=[parameters.Category.neutronics],
         )
 
-        pb.defParam("powerDecay", units=units.WATTS, description="Total decay power")
```

Comment:
I see this parameter being used in two different repos downstream.

Have you dealt with that? Opened PRs to fix those?

Reply:
yes, I will reach out offline